README.md

```diff
@@ -165,15 +165,43 @@ For easy deployment, we've created an official Docker container. There are two v
 
 The default one will run as `root`:
 
+> [!WARNING]
+> Do not use tag `latest` as this is not the latest released version. Do not use WatchTower using this tag as. User an actual version tag until transfert supports major or minor version tags.
+
 ```bash
 docker run --publish 8080:8080 dutchcoders/transfer.sh:latest --provider local --basedir /tmp/
 ```
 
+### No root
+
+The `-noroot` tags setup container images to run least priviledge to reduce the attack surface when an application is compromised.
```

Comment:
not sure about the wording here... "setup"? more like "indicates image builds that run with least priviledge"